.github/workflows/pr_e2e_tests.yml

```diff
@@ -0,0 +1,51 @@
+name: Pull request E2E tests
+
```

Comment:
why are you ignoring my previous PR feedback here?

Please either say you disagree with the feedback or agree with it but don't ignore it.

Reply:
I did not ignore your pervious feedback
This draft pr was never meant to be reviewed
I was using this PR to test solutions to an issue I linked in our slack
I agree with the changes you suggested and they will be there in the final version